Add Waterway Access Point preset and add fields to Slipway preset

quincylvania opened the pull request:
These are all well-established tags.

- https://wiki.openstreetmap.org/wiki/Tag:waterway%3Daccess_point
- https://wiki.openstreetmap.org/wiki/Key:canoe
- https://wiki.openstreetmap.org/wiki/Key:boat
- https://wiki.openstreetmap.org/wiki/Key:motorboat
- https://wiki.openstreetmap.org/wiki/Key:camping
- https://wiki.openstreetmap.org/wiki/Key:swimming
- https://wiki.openstreetmap.org/wiki/Key:trailer
- https://wiki.openstreetmap.org/wiki/Key:snowmobile

quincylvania committed:
Add Waterway Access Point preset and add field to Slipway preset

quincylvania commented on data/presets/leisure/slipway.json:
```diff
@@ -2,16 +2,41 @@
+        "reservation",
         "seamark/type",
+        "seasonal",
+        "shower",
```
Public showers are sometimes available at boat launches where swimming is popular. People will take the boat out, swim in a lake, and then shower upon returning to shore. Shower access is also important for people on overnight canoe trips that want to know what facilities are available at a given launch point.

matkoniecz commented on data/presets/waterway/access_point.json:
```diff
@@ -0,0 +1,27 @@
+{
```
probably icon distinct from one used for unrecognised tags would make sense, but I have no idea for anything

tyrasd replied:
maybe the coastline icon would fit?

<img width="82" height="66" alt="image" src="https://github.com/user-attachments/assets/fa2deac1-c7fe-4da4-b851-d0a1c61e4893" />

matkoniecz reviewed (approved):
cautious approval as I am not an expert on topic or tagging here

tyrasd reviewed (commented):
I would reuse the options from the `access_simple` field in all of the boat-specific access fields (and add `"autoSuggestions": false`). That way, the options are translated and recognizable. Or was there a specific reason to have them as generic combo fields that even allows free user input?

tyrasd committed:
add an icon for waterway=access_point

tyrasd committed:
move "tidal" field to moreFields

it's not a very common tag used in combination, and does not apply everywhere, so it's sufficient to keep it as an optional field only.

k-yle commented on data/fields/canoe.json:
```diff
@@ -0,0 +1,5 @@
+{
+    "key": "canoe",
+    "type": "combo",
+    "label": "Canoe Access"
```
a bit late, but [the defintion of this tag](http://osm.wiki/Key:canoe) says:

> `canoe=*` is the [access](https://wiki.openstreetmap.org/wiki/Access) tag for canoes, kayaks, and similar paddlecraft.

Would it make sense to rename this label to _"Paddlecraft Access"_ instead of just  _"Canoe Access"_ ? 

I realise that paddling in the inland US waterways is rather different to my own experience, so perhaps it's fine as-is? (given that these are `en-US` labels)

or we ignore this topic and revisit it after https://github.com/openstreetmap/iD/pull/12011 ?

quincylvania replied:
> Would it make sense to rename this label to "Paddlecraft Access" instead of just "Canoe Access" ?

This would probably be better, though "paddlecraft" isn't really a common word and could be confused with pedal boats or paddle wheel ships. I usually say "Canoe & Kayak" in this context but that still leaves out rafts and rowboats.

There has actually been an uptick in `kayak` and `sea_kayak` tagging lately, perhaps under the false assumption that `canoe` doesn't cover these.

matkoniecz replied:
maybe new, less misleading tag can be invented?
